packages/components/schema.graphql

```diff
@@ -1,26 +1,42 @@
-"""Exposes a URL that specifies the behaviour of this scalar."""
```

Comment:
All comments were updated. I think that's an IDE configuration issue. I guess you didn't change this in propose, so I would revert these unneeded changes.